@xen-orchestra/lite/src/locales/fr.json

```diff
@@ -31,7 +32,7 @@
   "hosts": "Hôtes",
   "language": "Langue",
   "last-week": "Semaine dernière",
-  "following-hosts-unreachable":"Les hôtes suivants sont inaccessibles",
+  "following-hosts-unreachable": "Les hôtes suivants sont inaccessibles",
```

Comment:
Remove and check all unnecessary changes in this file and in `en.json`.

Reply:
We can remove it for this PR, but we do need to fix it, though, since I'm guessing the fixes were made by Prettier and it's going to come back in every PR if we don't fix it.